odl/contrib/shearlab/README.md

```diff
@@ -0,0 +1,15 @@
+# Shearlab.jl
+**Original contribution by:** @arsenal9971  
```

Comment:
I realized in another PR that this will not automatically generate a link to your GH profile, so you may want to create that using `[@arsenal9971](https://github.com/arsenal9971)` notation